minid_client/minid.py

```diff
@@ -85,15 +89,16 @@ def main():
                                 checksum,
                                 args.email if args.email else config["email"],
                                 args.code if args.code else config["code"],
-                                locations, args.title, args.test, args.content_key)
+                                locations, args.title, args.test, args.content_key,
+                                args.globus_auth_token)
     elif args.update:
         if entities is None:
             print("No entity found to update. You must use a valid minid.")
             return
         elif len(entities) > 1:
             print("More than one minid identified. Please use a minid identifier")
         else:
-            entity = entities.values()[0]
+            entity = list(entities.values())[0]
```

Comment:
Not strictly related to this pull request, but this will throw an error in python 3 due to new 
 `dict_values` objects not supporting indexing.